corehq/motech/repeaters/utils.py

```diff
@@ -1,11 +1,11 @@
 from collections import OrderedDict
 
-from django.conf import settings
-
 from dimagi.utils.modules import to_function
 
+from corehq.motech.repeaters.const import REPEATER_CLASSES
+
 
 def get_all_repeater_types():
     return OrderedDict([
-        (to_function(cls, failhard=True).__name__, to_function(cls, failhard=True)) for cls in settings.REPEATERS
```

Comment:
Should this be communicated? Is it likely that anyone is hosting themselves and is overriding REPEATERS in localsettings?

Reply:
Very very unlikely. They can't add new classes (unless they've cloned the repo and written their own repeater classes), and if they didn't want to use the existing classes it would be easier for them just to not create any instances.

On my to-do list is to register repeater classes -- the way signal listeners work -- instead of using a list of class names.